app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -103,6 +103,30 @@
     margin: 1.5rem 0;
   }
 
+  .life-stage-nav {
+    background-color: #54853f;
+    overflow-x: auto;
+    overflow-y: hidden;
+
+    &.bar-header {
+      height: auto;
+    }
+
+    ul.chosen-stages {
+      margin: -1rem 0;
+      white-space: nowrap;
+
+      p {
+        color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.